cmake_modules/BuildUtils.cmake

```diff
@@ -182,13 +183,7 @@ function(ADD_ICEBERG_LIB LIB_NAME)
       target_include_directories(${LIB_NAME}_static PRIVATE ${ARG_PRIVATE_INCLUDES})
     endif()
 
-    if(MSVC_TOOLCHAIN)
-      set(LIB_NAME_STATIC ${LIB_NAME}_static)
-    else()
-      set(LIB_NAME_STATIC ${LIB_NAME})
-    endif()
-
-    set_target_properties(${LIB_NAME}_static PROPERTIES OUTPUT_NAME ${LIB_NAME_STATIC})
+    set_target_properties(${LIB_NAME}_static PROPERTIES OUTPUT_NAME ${LIB_NAME})
```

Comment:
I removed this because I encountered an error on Windows. It complained that `iceberg_arrow.dll` had referenced `iceberg_arrow.lib` but actually it was named `iceberg_arrow_static.lib`.